common/cache/domainCache.go

```diff
@@ -300,3 +308,33 @@ func (entry *DomainCacheEntry) GetDomainNotActiveErr() error {
 	}
 	return errors.NewDomainNotActiveError(entry.info.Name, entry.clusterMetadata.GetCurrentClusterName(), entry.replicationConfig.ActiveClusterName)
 }
+
+// ContentEqual return true if 2 domain cache entry have the same content
+func ContentEqual(this *DomainCacheEntry, that *DomainCacheEntry) bool {
+	// we do not check the mutex or expiry
+	if *this.info != *that.info {
```

Comment:
I thought you have a domain update version which can be used to check if the content is exactly the same?

Reply:
that was used to deal with conflict, but that can do the work, updating